command-reference.md

```diff
@@ -8,7 +8,7 @@ next: /rubygems-org-api
 
 <em class="t-gray">What each `gem` command does, and how to use it.</em>
 
-This reference was automatically generated from RubyGems version 3.1.4.
+This reference was automatically generated from RubyGems version 3.0.2.
```

Comment:
Sound like we're downgrading the help output and you need to generate it with the same version?

Reply:
Sorry, I give up. The rebase was the last thing I was going to do with this PR. Take the code if you want, do with it whatever you want, I'm done.

I actually tried to checkout v3.1.4 in `../rubygems` and run `rake command_guide`, but it crashes and I'm not going to dive into debugging.

Reply:
@deivid-rodriguez should I take a look?

Reply:
Yes, @simi, sure!

Thanks for getting this stated @igneus. We don't want to downgrade all guides to document an older version just to fix a whitespace issue, but the current state of this PR is very helpful to get this properly fixed.

Reply:
Something else I noticed is that our release tasks are supposed to automatically update this, but they are skipped on any branches other than master since https://github.com/rubygems/rubygems/commit/c733c95cd3902ffd82bad9ef30d562d4ba19224a. I think once we fix this issue we should revert that commit since although it probably made sense with the release strategy at the time, updating the guides automatically does play nice with our current strategy.

Reply:
@simi I tried running `rake command_guide` against the current version and it worked just fine, so nothing to fix here :). See #280. After we merge that, we can rebase this one again and it should be ready.

Reply:
Done!